src/refactor_nrepl/middleware.clj

```diff
@@ -35,6 +35,11 @@
   `(with-errors-being-passed-on ~transport ~msg
      (transport/send ~transport (response-for ~msg ~(apply hash-map kv)))))
 
+(defn format-response [{:keys [format] :as msg} response]
+  (if (= format "none")
```

Comment:
I think we should switch this around and do `(= format "edn")` so people aren't left wondering what exactly `none` means.